packages/valory/skills/abstract_round_abci/tests/test_base.py

```diff
@@ -18,6 +18,9 @@
 # ------------------------------------------------------------------------------
 
 """Test the base.py module of the skill."""
+
+# pylint: skip-file
+
```

Comment:
Why skip the whole file?

Reply:
Because we weren't using pylint on tests till now, and now it's just a pile of errors. Right now we can just port the tests and work around with refactoring and linter related issues laters.